js/imgur.js

```diff
@@ -45,14 +77,23 @@ weechat.factory('imgur', ['$rootScope', function($rootScope) {
         fd.append("image", base64img); // Append the file
         fd.append("type", "base64"); // Set image type to base64
 
+        // Comment the next line if you don't want to upload your pics into a folder.
```

Comment:
This comment doesn't make sense:
- users don't change the code
- if you don't want to upload your pics into a folder, just don't set the album hash.

Reply:
Corrected!